src/perform_step/exponential_rk_perform_step.jl

```diff
@@ -665,6 +666,69 @@ function perform_step!(integrator, cache::Exp4Cache, repeat_step=false)
   # integrator.k is automatically set due to aliasing
 end
 
+function perform_step!(integrator, cache::EPIRK4s3AConstantCache, repeat_step=false)
+  @unpack t,dt,uprev,f,p = integrator
+  A = f.jac(uprev, p, t)
```

Comment:
The EPIRKs should probably use the standard route for grabbing Jacobians, or everything should be updated at the same time.

Reply:
It's fine, keep it. https://github.com/JuliaDiffEq/DiffEqBase.jl/pull/113 makes DiffEqFunction-based ODEProblems the default, so this'll be the new style.